scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py

```diff
@@ -200,6 +200,28 @@ def maybe_float(string):
     default=None,
 )
 parser.add_argument("--ylabel", type=str, help="Label for y axis", default=None)
+
+parser.add_argument(
+    "--blocks",
+    dest="blocks",
+    default=True,
+    action="store_true",
+    help="Whether or not to plot block boundaries. Default is true.",
+)
+parser.add_argument(
+    "--no-blocks",
+    dest="blocks",
+    action="store_false",
+    help="Whether or not to plot block boundaries. Default is true.",
+)
```

Comment:
I haven't come across this pattern yet and the resulting help text
```
  --blocks              Whether or not to plot block boundaries. Default is true.
  --no-blocks           Whether or not to plot block boundaries. Default is true.
```
confused me a little.

Are both cases really needed or is one argument enough. What happens if I provide both? Which one takes precedence?

In principle, I'd be in favor of a single argument only.
If both arguments are required for some reason, I'd be great to get more specific help messages, maybe sth like.
```
  --blocks              Plot block boundaries (default)
  --no-blocks           Disable plotting block boundaries.
```